open_samus_returns_rando/pickup.py

```diff
@@ -262,10 +286,7 @@ def count_dna(lua_scripts: LuaEditor, pickup_object: BasePickup):
         lua_scripts.add_dna(scenario)
 
 def patch_pickups(editor: PatcherEditor, lua_scripts: LuaEditor, pickups_config: list[dict], configuration: dict):
-    all_pkgs = editor.get_all_level_pkgs()
-    editor.add_new_asset("actors/items/randomizer_powerup/scripts/randomizer_powerup.lc", b'', lua_pkgs(all_pkgs))
-    # FIXME: Why it doesn't work when we directly add it to the pkgs?
-    # Addition: Tested => It seems like this file is read fro RomFS but why not from pkg?
+    editor.add_new_asset("actors/items/randomizer_powerup/scripts/randomizer_powerup.lc", b'', [])
```

Comment:
This still looks a little bit weird but some lua files were not read from pkg files, so I decided to not add our own files to the pkgs at all.